e2e/payment/cancel.spec.ts

```diff
@@ -0,0 +1,42 @@
+import MercadoPago, { Payment } from '@src/index';
+import { config } from '../e2e.config';
+import type { PaymentCreateData } from '@src/clients/payment/create/types';
+
+describe('IT, cancel', () => {
+	test('should cancel Payment and match response object ', async () => {
+		const client = new MercadoPago({ accessToken: config.access_token, options: { timeout: 5000 } });
+		const payment = new Payment(client);
+
+		const paymentBody = createPayment();
+		const paymentCreate = await payment.create(paymentBody);
+		expect(paymentCreate).toHaveProperty('id');
+
+		const cancelation = await payment.cancel({ id: paymentCreate.id });
+		expect(cancelation).toHaveProperty('id', paymentCreate.id);
+		expect(cancelation).toHaveProperty('status', 'cancelled');
+	});
+
+	function createPayment(): PaymentCreateData {
```

Comment:
This doesn't need to be a function, it just needs to be a common variable., Do the same for all occurrences

```
const createPaymentBody = {
			body: {
				additional_info: {
					items: [
						{
							id: 'MLB2907679857',
							title: 'Point Mini',
							quantity: 1,
							unit_price: 58.8
						}
					]
				},
				payer: {
					email: 'test_user_123@testuser.com',
				},
				transaction_amount: 110.00,
				installments: 1,
				payment_method_id: 'pix',
			}
	}
```